README.md

```diff
@@ -69,34 +61,17 @@ as much as possible.
 
 ### Pull requests
 
-We welcome pull requests for this project and are always looking forward to
-review them.
-
-You can these steps to quickly submit a pull request.
-
-1. Fork the repository.
-2. Branch out to `a-meaningful-branch-name`.
-3. Make your changes and commit them along with a good summary of the changes as
-   the commit message. Refer to the [next subheading](#commit-message-guidelines)
-   for the commit message guidelines.
-3. Submit your pull request describing the changes introduced in the PR
-   description.
-4. Get your pull request merged!
+Diffmin is currently in its rudimentary stages so we are not expecting a pull
+request from outside contributors as there are not many features to work on as
+of now. We will update the instructions when we feel that we have enough issues
+to tackle. In that case, we would encourage contributions from anyone.
 
-### Commit message guidelines
+Internally, we do follow some guidelines before submitting a pull request.
```

Comment:
I am not sure if this is important to mention here. But I did it just to take a note of this formally.